server/api/cart.js

```diff
@@ -30,4 +30,63 @@ router.get('/', async (req, res, next) => {
   }
 })
 
-//for the view past orders get, use findall and no status validator
+router.post('/:id', async (req, res, next) => {
+  const id = req.params.id
+
+  //find record matching id
+  const foundRecord = await Record.findOne({
+    where: {
+      id: id
+    }
+  })
+
+  //create new row in Record
+  //!!THIS IS WHERE WE WOULD AFFECT QUANTITY - DOES THE RECORD ALREADY EXIST IN CART? IF SO, SIMPLY CHANGE QUANTITY, DO NOT ADD WHOLE NEW RECORDORDER
```

Comment:
First we need to find the 'pending' cart with matching userId === req.user.id or create a new cart for user. Then check the database for RecordOrders with the recordId == id and orderId === cart.id. If we find the recordOrder we just quantity++. The following code will create a new RecordOrder.